.github/workflows/ci.yml

```diff
@@ -155,8 +155,6 @@ jobs:
           pypi_token: ${{ secrets.PYPI_TOKEN }}
 
   build_wheels:
-    needs: [release]
```

Comment:
Let’s keep this but limit to when an actual release is made and not a test release